cmd/manager/config.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"github.com/spf13/pflag"
+)
+
+const (
+	// Change below variables to serve metrics on different host or port.
+	defaultTmpDir = "/tmp"
+
+	//Kops
+	defaultKopsStateStore     = "s3://kops.state.seizadi.infoblox.com"
+	defaultKopsClusterDnsZone = "soheil.belamaric.com"
+	defaultSSHKey             = "kops.pub"
+	defaultKopsContainer = "soheileizadi/kops:v1.0"
+	defaultKopsKubeDir = "/kube"
```

Comment:
remove the '/' from the default so it matches what the kops functions are expecting